central/group/datastore/internal/store/bolt/store_test.go

```diff
@@ -0,0 +1,311 @@
+package bolt
```

Comment:
Despite what GitHub shows, this is not a new file. It's just moved into the bolt folder and extensively modified.